types/dist/monero_wallet_full.d.ts

```diff
@@ -0,0 +1,5 @@
+export = monero_javascript;
```

Comment:
types/src and types/dist exist. I'll need to understand how to generate the source types and what modifications are necessary. I assume files within dist are 100% auto generated.

Reply:
To generate the initial typedefs, use the following command:

`npx -p typescript tsc src/*.js --declaration --allowJs --emitDeclarationOnly --outDir types`

After that each entry in index.d.ts needs to be changed to match the format:

  import * as MoneroWalletKeys from "./src/main/js/wallet/MoneroWalletKeys";
  export import MoneroWalletKeys = MoneroWalletKeys;

...with the exception of functions, ie


The @types/dist folder may actually be completely unnecessary. We'd have to test it with the folder removed to be sure.